From 1f3222b5f07c66005c2007c963cc833bf2a456b3 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Daniel=20Rodri=CC=81guez=20Troitin=CC=83o?=
 <danielrodriguez@fb.com>
Date: Thu, 11 Oct 2018 18:24:55 -0700
Subject: [#19846] Disable powl test in Android AArch64.

Android AArch64 seems to define long double as 128 bits floats.
Swift doesn't support that size, so Clang Importer doesn't import the
function at all.

Disable the function usage in the tests to avoid an error while
compiling the test file.
---
 test/ClangImporter/cfuncs_parse.swift | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/test/ClangImporter/cfuncs_parse.swift b/test/ClangImporter/cfuncs_parse.swift
index e5e2cb1d4bd..d7264c112fc 100644
--- a/test/ClangImporter/cfuncs_parse.swift
+++ b/test/ClangImporter/cfuncs_parse.swift
@@ -57,9 +57,13 @@ func test_pow() {
   pow(1.5, 2.5)
 }
 
+#if !os(Android) || !arch(arm64)
+// long doubles in Android AArch64 are 128 bits, which is not supported by
+// Swift, so don't test this.
 func test_powl() {
   powl(1.5, 2.5)
 }
+#endif
 
 func test_puts(_ s: String) {
   _ = s.withCString { puts($0) + 32 };
